client/src/test/java/io/pravega/client/stream/impl/SerializationTest.java

```diff
@@ -155,7 +156,7 @@ public void testReaderGroupUpdates() throws Exception {
         verify(serializer, new RemoveReader(createString(), createSegmentToLongMap()));
         verify(serializer, new ReleaseSegment(createString(), createSegment(), r.nextLong()));
         verify(serializer, new AcquireSegment(createString(), createSegment()));
-        verify(serializer, new UpdateDistanceToTail(createString(), r.nextLong()));
+        verify(serializer, new UpdateDistanceToTail(createString(), r.nextLong(), createSegmentRangeMap()));
```

Comment:
we should add tests for both the revisions right?

Reply:
Done